templates/vhost/_file_header.epp

```diff
@@ -0,0 +1,43 @@
+# ************************************
+# Vhost template in module puppetlabs-apache
+# Managed by Puppet
+# ************************************
+<%= [$comment].flatten.map |$c| { "# ${c}" }.join("\n") -%>
+<%- if $mdomain { -%>
+
+  <%- if type($mdomain) =~ Type[String] { -%>
```

Comment:
I think `type()` and `Type` aren't needed
```suggestion
  <%- if $mdomain =~ String { -%>
```

Reply:
Done